src/pages/User.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import logo from '../CodeKingTrans.png';
+import Navbar from './Navbar';
+import '../App.css';
+
+function UserCard({ user }){
+    return (
+        <div className='userCard'>
+            <img alt='profile logo' src='https://cdn-icons-png.flaticon.com/512/12225/12225773.png'></img>
+            <h2>{user.firstName + ' ' + user.lastName}</h2>
+        </div>
+    );
+}
+
+
+function User() {
+    const [users, setUsers] = useState([]);
+
+    useEffect(() => {
+        async function getData(){
+            const response = await fetch("https://disc-assignment-5-users-api.onrender.com/api/users")
+            const json = await response.json();
+            setUsers(json);
+        }
+        getData();
+    });
+    
+
+    return (
+        
+        <div className="App">
+            <Navbar />
+            <div className='backdrop'>
+                <img alt='Code King Logo' src={logo}></img>
+                <div className="userList">
+                    <div className='userListTitle'>
+                        <h1>Friends:</h1>
+                    </div>
+                    {users.map((user) => (
+                        <UserCard key={user.id} user={user} />
+                    ))}
+                </div>
+            </div>
+            
+            <div className='footer'>
+                <img alt='instagram logo' src='https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Instagram_logo_2016.svg/2048px-Instagram_logo_2016.svg.png'></img>
```

Comment:
Something you can do to declutter your code here is using a variable to store these super long image links.
Outside of your components, make a global variable
```
const ig_logo = "image url blah blah"
```
Then, you can do
```
<img src={ig_logo} />
```

This just makes your code slightly cleaner